extensions/azurecore/src/account-provider/auths/azureAuthCodeGrant.ts

```diff
@@ -111,7 +112,7 @@ export class AzureAuthCodeGrant extends AzureAuth {
 		} catch (err) {
 			const msg = localize('azure.serverCouldNotStart', 'Server could not start. This could be a permissions error or an incompatibility on your system. You can try enabling device code authentication from settings.');
 			vscode.window.showErrorMessage(msg);
-			console.dir(err);
+			Logger.error(JSON.stringify(err));
```

Comment:
Why stringify this one but not any of the others? Errors generally print out nicely in my experience (including stack trace and whatnot) without needing to stringify

Reply:
Because the error with this one is stacked multiple levels deep (Just node webserver things...)